metadata-ingestion/src/datahub/ingestion/source/superset.py

```diff
@@ -251,15 +256,20 @@ def emit_dashboard_mces(self) -> Iterable[MetadataWorkUnit]:
 
         while current_dashboard_page * PAGE_SIZE <= total_dashboards:
             dashboard_response = self.session.get(
-                f"{self.config.connect_uri}/api/v1/dashboard",
+                f"{self.config.connect_uri}/api/v1/dashboard/",
                 params=f"q=(page:{current_dashboard_page},page_size:{PAGE_SIZE})",
             )
+            if dashboard_response.status_code != 200:
+                logger.warning(
+                    f"Failed to get dashboard data: {dashboard_response.text}"
+                )
+            dashboard_response.raise_for_status()
```

Comment:
will this break full connector or just for these dashboards?

Reply:
same as above